CMSgov/mint-app
# NOREF/ch unrelated findings 1

## Description
This PR addresses some unrelated findings that were discovered during Change History. Please reference the [change history review document](https://docs.google.com/document/d/17zoYGYf3nT6i_Cj6svLE_Qvv2GJLNnCbZRD5Vvt6B0I/edit#heading=h.w1c00sz7l73q).
[21.](https://docs.google.com/document/d/17zoYGYf3nT6i_Cj6svLE_Qvv2GJLNnCbZRD5Vvt6B0I/edit#heading=h.dak9fu4a9afh) Finding basics --> model Basics
[22](https://docs.google.com/document/d/17zoYGYf3nT6i_Cj6svLE_Qvv2GJLNnCbZRD5Vvt6B0I/edit#heading=h.6gz78apldcxu). Daily Digest Text Casing
   - Note, this could be adjusted even more, eg translating the table name with a shared mapping. I'm doing the simplest fix for now
   
  Outside Scope
  [1](https://docs.google.com/document/d/17zoYGYf3nT6i_Cj6svLE_Qvv2GJLNnCbZRD5Vvt6B0I/edit#heading=h.h96ztj9241lu) Note field for what is agreement type conditional status

[3](https://docs.google.com/document/d/17zoYGYf3nT6i_Cj6svLE_Qvv2GJLNnCbZRD5Vvt6B0I/edit#heading=h.w1c00sz7l73q) Email link for solutions to POC not going to filter view



## How to test this change

1.  Look the review document (linked above in each info blurb)
2. Follow the steps to reproduce the issue
3. confirm that the issue is no longer occuring.


Please also verify unit tests pass, and the code changes are as expected.

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
